blog_system/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-75dk=yot+&l6u_lr+$*684z8o)0k6ahrq9g%h@%@r#^w$8d6nm'
```

Comment:
Security issue: Your secret key is exposed. This is a potential security risk. Consider storing secrets as environment variables or in a secure secret manager. Make sure not to expose your secret key in your code or version control system.